viper/parser/parser.py

```diff
@@ -693,7 +693,7 @@ def pack_args_by_32(holder, maxlen, arg, typ, context, placeholder):
             if input.typ.maxlen > typ.maxlen:
                 raise TypeMismatchException("Data input bytes are to big: %r %r" % (input.typ, typ))
             if arg.id in context.vars:
-                size = context.vars[arg.id].size
+                size = input.typ.maxlen
```

Comment:
we should really rename input actually, since it's a python built-in function.

Reply:
💯